canonical-web-and-design/multipass.run
# Build new homepage

Update the homepage to match the copy doc and new design.

## QA
- Run `./run`
- Open http://0.0.0.0:8026/
- Check the content matches [the copy doc](https://docs.google.com/document/d/1uI5hsuhe5VUpFSSiQas6FVB2F7RhlbNXcr5Pvr_Lm2c/edit#)
- Check it matches [the design](https://user-images.githubusercontent.com/9105999/56364510-9fa0d200-61e6-11e9-8e30-9434a9a2e8ef.png)
- Check on different screens sizes

## Screenshot
![Screenshot_2019-04-23 Multipass - Orchestrates virtual Ubuntu instancess](https://user-images.githubusercontent.com/1413534/56585150-2a157700-65d5-11e9-9414-e9448443c679.png)

Fixes https://github.com/canonical-web-and-design/multipass.run/issues/17
Fixes https://github.com/canonical-web-and-design/multipass.run/issues/13
